templates/CRM/Custom/Form/Edit/CustomField.tpl

```diff
@@ -51,7 +51,7 @@
     <td class="html-adjust">
       {$formElement.html}&nbsp;
       {if $element.data_type eq 'File'}
-        {if $element.element_value.data}
+        {if array_key_exists('element_value', $element) && $element.element_value.data}
```

Comment:
There was some suggestion at https://github.com/civicrm/civicrm-core/pull/26964#issuecomment-1655904099 that sometimes this is not an array, but I think what it is is that the type of element_value depends on the type of custom field, so for file it (should be) an array, and given there was no confirmation over there and the original bug there seems unreproducible, I think this is ok.

Reply:
thanks @demeritcowboy